---
gate_depth: -1.5
gate_search_depth: -0.7
target_gate_link: "gate_blue_arrow_link"

red_buoy_radius: 2.2
red_buoy_depth: -0.7

torpedo_map_radius: 1.5
torpedo_map_depth: -1.3

bin_task_depth: -0.7

octagon_depth: -1.0

state_map:
  init: INITIALIZE
  gate: NAVIGATE_THROUGH_GATE
  buoy: NAVIGATE_AROUND_RED_BUOY
  torpedo: NAVIGATE_TO_TORPEDO_TASK
  bin: NAVIGATE_TO_BIN_TASK
  octagon: NAVIGATE_TO_OCTAGON_TASK

full_mission_states: ['INITIALIZE', 'NAVIGATE_THROUGH_GATE', 'NAVIGATE_AROUND_RED_BUOY', 'NAVIGATE_TO_TORPEDO_TASK', 'NAVIGATE_TO_BIN_TASK', 'NAVIGATE_TO_OCTAGON_TASK']

test_states: ['init']
